### ISSUE TITLE
Make formatter optional

### ISSUE THREAD

**Issue by aserafin** [aserafin/grape_logging#23]
There are logger implementations that don't expose `.formatter. Make grape-logging compatible with them!

**Comment by guizmaii** [comment]
Maybe just add a `if` like that could solve the problem in `LoggerReporter#initialize` : 

```ruby
@logger.formatter = formatter || GrapeLogging::Formatters::Default.new if @logger.respond_to?(:formatter)
```

**Comment by aserafin** [comment]
Exactly my thinking :)
15.03.2016 3:23 PM "Jules Ivanic" <notifications@github.com> napisał(a):

**Comment by guizmaii** [comment]
I'm not so able to test but I pushed you the modification #24 ;)

**Comment by guizmaii** [comment]
Actually it's not `@logger.respond_to?(:formatter)` but `@logger.respond_to?(:formatter=)` that we should test, I think.

**Comment by aserafin** [comment]
Merged #24 which should solve the issue. Thanks @guizmaii :+1:
